modules/ort/src/test/java/org/eclipse/sw360/antenna/ort/workflow/analyzers/OrtResultAnalyzerTest.java

```diff
@@ -161,6 +158,18 @@ public void testMapCoordinates() throws URISyntaxException, IOException {
 
     }
 
+    @Test
+    public void testAnalyzerOrtDataToArtifact() throws IOException, URISyntaxException {
+        List<Artifact> artifacts = init("single_package.yml");
+
+        assertThat(artifacts.stream().
```

Comment:
Usually the dots are in the next line:
```suggestion
        assertThat(artifacts.stream()
                           .map(artifact -> artifact.askForGet(ArtifactHomepage.class))
```